### PR TITLE
299395 sending/receiving of messages

### PR DESCRIPTION
**Pull request by kiranpal-atos**
Test Evidence: 
image

### CODE REVIEW

File: cypress/utils/receiveMessage.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const { ServiceBusClient } = require('@azure/service-bus');
+
+// Connection string for your Azure Service Bus namespace
+const connectionString = 'Endpoint=sb://sndffcinfsb1001.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=XAs4qFZfUzO1aURxztvK/q7nTxx74yNQB+ASbFzFhhg=';
```
Comment (GodsonLeigh) [inline cypress/utils/receiveMessage.js:4]: Should this connection string be hidden and set somewhere else? I am not sure of the scope of the test files or how they sit within what we usually do for lines like this.

File: cypress/utils/sendMessageQueue.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const { ServiceBusClient } = require('@azure/service-bus');
+
+const connectionString = 'Endpoint=sb://sndffcinfsb1001.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=XAs4qFZfUzO1aURxztvK/q7nTxx74yNQB+ASbFzFhhg=';
```
Comment (GodsonLeigh) [inline cypress/utils/sendMessageQueue.js:3]: same query as the last one regarding the full string being used in this way.

**Review by GodsonLeigh — COMMENTED** [review]
Two comments just to query the use of the full connection string in code

File: cypress/utils/receiveMessage.js (hunk 1 above)
Comment (kiranpal-atos) [reply cypress/utils/receiveMessage.js]: updated this to use a .env file

File: cypress/utils/sendMessageQueue.js (hunk 2 above)
Comment (kiranpal-atos) [reply cypress/utils/sendMessageQueue.js]: updated this to use a .env file

**Review by GodsonLeigh — APPROVED** [review]
approve gif

**Pull request merged**